content/docs/intro/vs/cloud_template_transpilers.md

```diff
@@ -8,65 +8,65 @@ menu:
 ---
 
 Because of [the challenges of writing raw YAML/JSON by hand]({{< relref "cloud_templates.md" >}}), two notable
-projects exist to compile higher level languages into AWS CloudFormation YAML/JSON templates:
+projects exist to compile higher-level languages into AWS CloudFormation YAML/JSON templates:
 
 * **Troposphere**: a community-led open source project created in 2013
 * **AWS Cloud Development Kit (CDK)**: an AWS Labs project created in 2018
 
-Similar to Pulumi, these projects let you author infrastructure as code using general purpose languages like TypeScript,
+Similar to Pulumi, these projects let you author infrastructure as code using general-purpose languages like TypeScript,
 JavaScript, and Python. Unlike Pulumi, however, whose open source engine understands these languages, a _transpiler_
-(a.k.a., [_source-to-source compiler_](https://en.wikipedia.org/wiki/Source-to-source_compiler)) translates this program
-into [AWS CloudFormation YAML/JSON]({{< relref "cloud_templates.md" >}}). This resulting markup file is then submitted
+a.k.a., [_source-to-source compiler_](https://en.wikipedia.org/wiki/Source-to-source_compiler), translates this program
+into [AWS CloudFormation YAML/JSON]({{< relref "cloud_templates.md" >}}). The resulting markup file is then submitted
 to the closed source AWS CloudFormation servers to provision infrastructure on AWS in the usual ways.
 
 ## Summary of Differences
 
 The transpiler approach gives you some of the benefits of Pulumi, with the following caveats:
 
-* Troposphere/CDK only support the AWS platform. Pulumi supports many clouds, including: major cloud platforms
-  such as Microsoft Azure, Google Cloud Platform, Kubernetes, and DigitalOcean; on-premises and hybrid technologies
-  such as VMWare vSphere and OpenStack; and online SaaS offerings like Cloudflare, Datadog, New Relic, and more.
+* Troposphere/CDK only supports the AWS platform. Pulumi supports many clouds, including _major cloud platforms_
```

Comment:
Troposphere and CDK are separate entities, is "supports" still correct? I would write "Troposphere and CDK only support"

Reply:
Yeah, I went back and forth on this. Sorry for the premature merge. I'll include this change on the next PR.